lib/locales/hu.ts

```diff
@@ -0,0 +1,938 @@
+    remigius_of_reims_bishop: 'Saint Remigius, Bishop',
+    richard_gwyn_martyr: 'Saint Richard Gwyn, Martyr',
+    richard_of_chichester_bishop: 'Saint Richard of Chichester, Bishop',
+    richardis_of_swabia_empress: 'Saint Richardis, Empress',
+    rita_of_cascia_religious: 'Saint Rita of Cascia, Religious',
+    robert_bellarmine_bishop: 'Saint Robert Bellarmine, Bishop and Doctor of the Church',
+    roch_gonzalez_alphonsus_rodriguez_and_john_del_castillo_priests: 'Saints Roch González, Alphonsus Rodríguez and John del Castillo, Priests and Martyrs',
+    roch_of_montpellier: 'Saint Roch',
+    romuald_of_ravenna_abbot: 'Saint Romuald, Abbot',
+    rosalie_jeanne_marie_rendu_virgin: 'Blessed Rosalie Rendu, Virgin',
+    rose_of_lima_virgin: 'Saint Rose of Lima, Virgin',
+    rose_of_lima_virgin_copatroness_of_the_philippines: 'Saint Rose of Lima, Virgin, Copatroness of the Philippines',
+    rose_philippine_duchesne_virgin: 'Saint Rose Philippine Duchesne, Virgin',
+    rupert_of_salzburg_and_virgilius_of_salzburg_bishops: 'Saints Rupert and Virgilius of Salzburg, Bishops',
+    salomea_of_poland_religious: 'Blessed Salomea of Poland, Religious',
+    sancha_of_portugal_and_mafalda_of_portugal_virgins: 'Blessed Sancha and Mafalda, Virgins',*/
+    sara_salkahazi_virgin: 'Boldog Salkaházi Sára szűz és vértanú',
+    /*scholastica_of_nursia_virgin: 'Saint Scholastica, Virgin',
+    sebastian_de_aparicio_religious: 'Blessed Sebastian de Aparicio, Religious',
+    sebastian_of_milan_martyr: 'Saint Sebastian, Martyr',
+    senan_of_inis_cathaigh_bishop: 'Saint Senan, Bishop',
+    seven_holy_founders_of_the_servite_order: 'The Seven Holy Founders of the Servite Order',
+    seven_martyred_nuns_from_the_franciscan_missionaries_of_mary: 'Seven Martyred Nuns from the Franciscan Missionaries of Mary',
+    severinus_of_noricum_monk: 'Saint Severinus of Noricum, Monk',
+    sharbel_makhluf_priest: 'Saint Sharbel Makhlūf, Priest',
+    shipwreck_of_saint_paul_apostle: 'Shipwreck of Saint Paul, Apostle',
+    sigismund_of_burgundy_martyr: 'Saint Sigismund, Martyr',
+    sigmund_felix_felinski_bishop: 'Saint Sigmund Felix Feliński, Bishop',
+    simon_and_jude_apostles: 'Saints Simon and Jude, Apostles',
+    simon_of_lipnica_priest: 'Saint Simon of Lipnica, Priest',
+    six_welsh_martyrs_and_companions: 'Six Welsh Martyrs and Companions',
+    sixtus_ii_pope_and_companions_martyrs: 'Saint Sixtus II, Pope, and Companions, Martyrs',
+    spyridon_of_trimythous_bishop: 'Saint Spyridon, Bishop',
+    stanislaus_kazimierczyk_priest: 'Saint Stanislaus Kazimierczyk, Priest',
+    stanislaus_kostka_religious: 'Saint Stanislaus Kostka, Religious',
+    stanislaus_of_szczepanow_bishop: 'Saint Stanislaus, Bishop and Martyr',
+    stanislaus_of_szczepanow_bishop_patron_of_poland: 'Saint Stanislaus, Bishop and Martyr, Patron of Poland',
+    stephen_i_of_hungary: 'Saint Stephen of Hungary',*/
+    stephen_the_first_martyr: 'Szent István első vértanú',
```

Comment:
According to [_Direktórium 2022_](http://tar.liturgia.hu/Naptar/DIR2022_JAV_2021-12-13.pdf#page=42), it should be `Szent István diakónus, első vértanú`. Again, if it is different from the Hungarian Roman Missal, we should prefer the one from the Roman Missal.